lib/resourceFactory.js

```diff
@@ -72,7 +69,7 @@ export function createReader({fsBasePath, virBasePath, project, excludes = [], n
 			fsBasePath,
 			virBasePath,
 			project,
-			excludes: normalizedExcludes
+			excludes: excludes
```

Comment:
Please use the shorthand. Key and variable name are the same.

Reply:
Done